frontend/src/styles/mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin standard_button {
+  height: 50px;
+  background: transparent;
+  border-radius: 8px;
+  border: 1px solid $contrast-color;
+  color: $contrast-color;
+  margin-top: 24px;
+  cursor: pointer;
+  font-size: 1rem;
```

Comment:
The text on the buttons seems quite small when compared to the size of the button. My suggestion is to increase the font-size to `1.125rem` (18px).

Reply:
✅